.drone.yml

```diff
@@ -82,6 +82,10 @@ steps:
         - failure
 
 trigger:
+  branch:
```

Comment:
this is notify pipeline

we need to update main pipeline (notify can be excluded as well, but less important)

Reply:
good catch